src/app/demos/navbar.component.html

```diff
@@ -22,11 +24,12 @@ <h6>Supra bar</h6>
             </a>
         </li>
         <li class="nav-item" ngbDropdown>
-            <button class="btn btn-link nav-link" id="dropdownLanguage" ngbDropdownToggle>EN</button>
-            <ul class="dropdown-menu-right" ngbDropdownMenu aria-labelledby="dropdownLanguage">
-              <li><a ngbDropdownItem href="#">FR</a></li>
-              <li><a ngbDropdownItem href="#">SP</a></li>
-            </ul>
+          <button class="btn btn-link nav-link" id="dropdownLanguage" ngbDropdownToggle>Languages</button>
+          <ul ngbDropdownMenu aria-labelledby="dropdownLanguage">
+            <li><a ngbDropdownItem aria-label="Version en anglais" lang="en" hreflang="en" href="#">EN</a></li>
```

Comment:
Write `aria-label`content in the target's language: _"English version"_, _"Versión en español"_.

And if possible, also add on the current language item:
- `aria-current="true"` (`true` and not `page`!).
- `active` class.